server/src/modules/rules/getter/jellyseerr-getter.service.ts

```diff
@@ -116,6 +116,8 @@ export class JellyseerrGetterService {
                     if (includesSeason) {
                       if (request.requestedBy?.userType === 2) {
                         userNames.push(request.requestedBy?.username);
+                      } else if (request.requestedBy?.userType === 3) {
```

Comment:
So it looks Emby linked accounts also populate jellyfinUsername, so we can support that too:

```suggestion
                      } else if (request.requestedBy?.userType === 3 || request.requestedBy?.userType === 4) {
```